enzoic/enzoic.py

```diff
@@ -537,22 +590,80 @@ def get_user_passwords(self, username: str, include_exposure_details: bool = Fal
         Returns a list of passwords that Enzoic has found for a specific user.  This call must be enabled for your
         account or you will receive a 403 rejection when attempting to call it.
         see https://docs.enzoic.com/enzoic-api-developer-documentation/api-reference/credentials-api/cleartext-credentials-api
-        :param username: The username you wish to receive the a list of passwords for.
+        :param username: The username you wish to receive a list of passwords for.
         :param include_exposure_details: Includes the details of the exposure the password was found in if True,
         :return:
         """
         # username needs to be converted to lowercase and url encoded
-        params = {
+        query_params = {
             "username": str(username).lower(),
             "includePasswords": 1,
```

Comment:
Not necessary to include the includePasswords param if the /cleartext-credentials endpoint is used.  This is implied, so we can drop this.